### DISCUSSION TITLE
Font Path Magick++ API access

### DISCUSSION THREAD

**Discussion by adityasharma-quark** [ImageMagick/ImageMagick#6585] [Help]
### ImageMagick version

7

### Operating system, version and so on

windows 11, x64 bit configuration intel i7

### Description

I am not sure if there is any Magick++ API for Drawing Fonts on an image which could take path of the font as an argument such that the path could be of private fonts folder (i.e different from the system's fonts folder). As I was learning from the documentation provided "Font Paths" was under development, so is there any API or it is currently in the developing phase.
Thankyou in advance!

**Comment by snibgo** [comment]
Just as in the CLI, you can use a filepath instead of a font name. For example (Windows):
```
    image.font( "C:\\MyDir\\myFonts\\times.ttf" ); 
```
